### DISCUSSION TITLE
HomeBridge?

### DISCUSSION THREAD

**Discussion by ArielZgo** [gcormier/megadesk#54] [Ideas]
Any possibility to add it to HomeBridge? Some bluetooth add-on? Or Possibility to connect it to Shelly 2.5? :)

**Comment by gcormier** [comment]
#11 talked about it a little bit. The issue is there is no space inside the stock enclosure.

I think the best bet would be a companion board that talks to megadesk over serial.  #12 added this and is now merged, but you will need to compile with the serial option enabled as it is disabled by default.
